### DISCUSSION TITLE
Error: Cannot find module 'node:path'

### DISCUSSION THREAD

**Discussion by brillout** [vikejs/vike#770] [Discussion]
hey guys. I explore vite-plugin-ssr. Do pnpm init vite-plugin-ssr , after pnpm install and pnpm run dev

Get error: 
> failed to load config from ....../tea-votary/vite.config.ts
> (node:47290) UnhandledPromiseRejectionWarning: Error: Cannot find module 'node:path'

How fix it?

**Comment by brillout** [comment]
Does your Node.js version support `node:path`?

**Reply by artskadin** [reply]
I used node v14.17.1. It seems that was the main problem. I tried node v18.15.0 and it's work.

But I have mac M1 and there were also some problems with the processor architecture because I used the terminal under rosetta, but in IDE used arm64. I advise to use it under the same architecture (I used arm64). That's work
